week8/src/Home.js

```diff
@@ -0,0 +1,42 @@
+import React,{useState,useEffect} from "react";
+import { Form } from "react-bootstrap";
+import MovieBox from "./MovieBox";
+
+function Home() {
+    const [movies, setMovies]= useState([]);
+    const [title, setTitle]= useState('');
+    const getMovie=()=>{
+        fetch(`https://api.themoviedb.org/3/search/movie?api_key=b7d68d57b175ae831b45672648c74d7b&query=`)
+        .then(res=> res.json())
+        .then(json => console.log(json))
+    }
+useEffect(()=>{
+    getMovie()
+    console.log(process.env.REACT_APP_API_KEY)
+},[])
+const searchMovie = async(e)=>{
+    e.preventDefault();
+    try{
+const url = `https://api.themoviedb.org/3/search/movie?api_key=b7d68d57b175ae831b45672648c74d7b&query=${title}`
```

Comment:
we can use a single BASE_URL constant and then append anything we need to it, instead of repeating the URL in multiple places